### PR TITLE
refactor(overview): refactor app overview when no release to match up…

### LINKED ISSUE #74: update no release view to match the new design
Screenshot 2019-08-19 at 12 40 04

### PR DESCRIPTION
**Pull request by Arinono**
Fixes #74

### CODE REVIEW

File: src/views/Dashboard/App/NoRelease.vue [hunk 1]
```diff
@@ -12,169 +9,318 @@
...
     SLogo,
-    SCode
+    SCode,
+    SButton
   }
 })
 export default class NoRelease extends Vue {
+  @Prop({
+    type: String,
+    default: 'app-jeanvaljeanjean-007'
```
Comment (JeanBarriere) [inline src/views/Dashboard/App/NoRelease.vue]: remove this

File: src/views/Dashboard/App/NoRelease.vue [hunk 2]
```diff
@@ -12,169 +9,318 @@
...
+  }) readonly name!: string | undefined
+
+  @InjectReactive('uuid') private readonly uuid!: string
+
+  private os: string | undefined = undefined
+  // private uuid: string | undefined = undefined
 
+  mounted () {
+    this.os = this.osAgent
```
Comment (JeanBarriere) [inline src/views/Dashboard/App/NoRelease.vue]: not needed, just call `osAgent` as a computed, no ?

File: src/views/Dashboard/App/NoRelease.vue (hunk 1 above)
Comment (Arinono) [reply src/views/Dashboard/App/NoRelease.vue]: Done

File: src/views/Dashboard/App/NoRelease.vue (hunk 2 above)
Comment (Arinono) [reply src/views/Dashboard/App/NoRelease.vue]: Needed because I override `this.os` in the template when a button is clicked

**Pull request merged**